.github/workflows/create_release.yml

```
@@ -0,0 +1,58 @@
+name: "🚀 Release on merge"
+
+on:
+  pull_request:
+    types: [closed]
```

Should we do filter by branch name here?
```suggestion
    types: [closed]
    branches:
    - 'releases**'
```

---

the branches is for the target branch (afaik), here we want to make sure the source branch is release/*